marc.py

```diff
@@ -618,9 +611,9 @@ def records(self, lane, annotator, start_time=None, force_refresh=False,
         # and Representation, but don't actually mirror it.
         if not mirror:
             storage_protocol = self.integration.setting(self.STORAGE_PROTOCOL).value
-            mirror = MirrorUploader.sitewide(self._db)
+            mirror = MirrorUploader.implementation(integration)
```

Comment:
It looks like `integration` is required, and if so, you shouldn't set a default of `None` in the method signature.

Reply:
This is looking good, but `integration` is too vague a term. How about `mirror_integration`?